ansible/examples/slave.yml

```diff
@@ -126,7 +126,7 @@
       when: ansible_os_family == "RedHat"
 
     - name: Install RPM requirements
-      yum:
+      package:
```

Comment:
is this change because the python-yum library doesn't exist? Or is it because it is now `dnf` and not `yum` ?

Reply:
It exists but is not installed and the default is `dnf`.  So it's package-manager agnostic.